tests/t_synonyms

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+SDCV="$1"
+TEST_DIR="$2"
+
+unset SDCV_PAGER
+
+RES=$($SDCV -n --data-dir "$TEST_DIR" foo | grep result)
```

Comment:
You could move repeated pattern into `sh function`, for example:

```sh
function test_word() {
    WORD="$1"
    echo "Test ${WORD}"
    RES=$($SDCV -n --data-dir "$TEST_DIR" "${WORD}" | grep result)
    if [ "result" != "$RES" ]; then
        echo "synonym for "${WORD}" should be result but was \"${RES}\""
        exit 1
    fi
}

test_word "foo"
test_word "bar"
test_word "test"
```

also to make possible code to run on machine with dictionaries with "test" word into it,
please add `-u "Test synonyms"` into `sdcv` arguments